.github/meta.yaml

```diff
@@ -73,7 +73,7 @@ outputs:
         - lime >=0.2.0.1
         - python >=3.8.*
         - imbalanced-learn >=0.9.1
-        - sktime <0.13.1,>=0.12.0
+        - sktime >=0.13.3
```

Comment:
The conda is currently failing as it looks like the [latest version of sktime](https://anaconda.org/conda-forge/sktime) that is available on conda is 0.12.1 when we need this version for pandas 1.5 support. If pandas 1.5 support isn't super high priority we might need to skip conda support for now.

Reply:
https://github.com/sktime/sktime/issues/2921 - related. Can you file an issue against sktime to release 0.13.3?